themes/root/templates/HelpTranslations/en/markdown.phtml

```diff
@@ -0,0 +1,67 @@
+<h1>Markdown and emojis</h1>
+
+<ul class="HelpMenu">
+  <li><a href="#markdown">Markdown</a></li>
+  <li><a href="#emojis">Emojis</a></li>
+</ul>
+
+<dl class="Content">
+  <dt><a id="markdown"></a>Markdown</dt>
+  <dd>
+    <p>
+      Markdown is a simple way to format text using symbols. You can add formatting to your text without needing
+      to learn complicated coding.
+      <b>Please see <a href="https://www.markdownguide.org/basic-syntax/" target="_blank">this guide</a>
```

Comment:
For accessibility, it's better to have more descriptive links -- I think you could change "this guide" to "the Markdown Guide" to retain the meaning while making the link more self-explanatory.